ajax/libs/weblas/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weblas",
+  "description": "GPU accelerated BLAS for node and the browser",
+  "filename": "weblas.js",
+  "keywords": [
+    "gpu",
+    "blas",
+    "gemm",
+    "matrix",
+    "multiply",
+    "neural network",
+    "deep learning"
+  ],
+  "author": "Waylon Flinn <waylonflinn@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/waylonflinn/weblas"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/waylonflinn/weblas.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "weblas.js",
+          "weblas.map.json"
```

Comment:
Let's use `**/*`.
For more details please see https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md